packages/forms/frontend/sirius-components-forms/src/propertysections/PropertySection.tsx

```diff
@@ -138,7 +138,7 @@ export const PropertySection = ({
       />
     );
   } else if (isLink(widget)) {
-    propertySection = <LinkPropertySection widget={widget} key={widget.id} />;
+    propertySection = <LinkPropertySection widget={widget} subscribers={subscribers} key={widget.id} />;
```

Comment:
It makes no sense to add support for the subscribers of the link widget since there are literally no way for a subscriber to appear on a link.